recipes-support/scripts/files/aero-get-version.py

```diff
@@ -31,7 +31,7 @@ def FPGA_version():
 		else:
 			lines = output.split('\n')
 			words = lines[4].split()
-			print words[1] + " " + words[2]
```

Comment:
not really. After this change I get 0xc0 as version, as it should be from the commit on the FPGA.